packages/lambdas/src/document-downloader.ts

```diff
@@ -72,7 +73,10 @@ export const handler = Sentry.AWSLambda.wrapHandler(
       },
       capture,
     });
-    const result = await docDownloader.download({ document, fileInfo });
+    const result = await executeWithNetworkRetries(
+      () => docDownloader.download({ document, fileInfo }),
+      { retryOnTimeout: true, initialDelay: 500, maxAttempts: 5 }
+    );
```

Comment:
this function calls getFileInfoFromS3 which already is wrapped with executeWithNetworkRetries, and it calls downloadDocumentFromCW which also has inside it a wrap of executeWithNetworkRetries

ditto on layers of wrapping

Reply:
done